### ISSUE TITLE
process.cpu.time_in_millis multiplied by 1M

### ISSUE THREAD

**Issue by shoggeh** [elastic/logstash#7239]
- Version: 5.4.0
- Operating System: Linux
- Sample Data / Steps to Reproduce:

It looks like `process.cpu.total_in_millis` time reported in monitoring API is erroneously  multiplied by 1,000,000. My logstash says it's been using ~14 days of CPU time every second. Value seems to be perfectly correct after dividing by 1M.

```
$ curl -s 'http://127.0.0.1:9600/_node/stats?timeout=1s&pretty' | grep -A1 cpu ; sleep 1 ; curl -s 'http://127.0.0.1:9600/_node/stats?timeout=1s&pretty' | grep -A1 cpu
    "cpu" : {
      "total_in_millis" : 454039020000000,
    "cpu" : {
      "total_in_millis" : 454043540000000,
```

**Comment by original-brownbear** [comment]
@shoggeh thanks for reporting this, yes this is a bug. We're apparently using https://docs.oracle.com/javase/7/docs/jre/api/management/extension/com/sun/management/OperatingSystemMXBean.html#getProcessCpuTime() to populate this, which is measured in `ns` so `1E6` as a factor makes perfect sense.

**Comment by original-brownbear** [comment]
Fixed in `5.5` and `6.0`, `5.4` fix outstanding

**Comment by suyograo** [comment]
Not super critical to backport to 5.4 so closing this. 5.5 will be out in a few weeks. thanks for the fix @original-brownbear
